Providers/nxOMSAutomationWorker/automationworker/worker/httpclient.py

```diff
@@ -114,18 +114,44 @@ def delete(self, url, headers=None, data=None):
         """
         pass
 
-
 class RequestResponse:
     """Encapsulates all request response for http clients. Will also deserialize the response when the raw response
     data is deserializable.
     """
+    @staticmethod
+    def check_if_service_unavailable_response_is_received(response_body):
+        SERVICE_UNAVAILABLE_STR = 'Service Unavailable'
+        HTML_HEADER_TAG_REGEX = '<h[0-9]+>(.*?)</h[0-9]+>'
+        HTML_BODY_TAG_REGEX = '<BODY>(.*?)</BODY>'
+        HTML_PARAGRAPH_TAG_REGEX = '<p>(.*?)</p>'
```

Comment:
Can we move this assignment outside as constants or something? Here objects will be created every time this method is run.

Reply:
I thought these are just strings whose scope is limited to just this method. Hence kept them inside this method

Reply:
Sure but these strings can be a part of the class definition.  No point of recreating them every time this method runs.